src/components/Game/onMessage.ts

```diff
@@ -339,13 +343,41 @@ export const onMessage_player = (
       let currentGameTurn = state.gameTurn;
       const boardCardInfo = message.showInfo.boardCardInfo;
       const isShowDown = boardCardInfo.every(x => x !== null);
+      const { winners, win_amount } = message; //eslint-disable-line @typescript-eslint/camelcase
+
+      // Log winners to hand history
+      const logWinners = (): void => {
+        // Log if there is a single winner
+        if (winners.length === 1) {
+          addToHandHistory(
+            `Player${winners[0] + 1} wins ${numberWithCommas(win_amount)}.`, //eslint-disable-line @typescript-eslint/camelcase
+            dispatch
+          );
+          // Log if the pot is split between multiple players
+        } else if (winners.length > 1 && winners.length < 10) {
+          const winnerList = arrayListToSentence(
+            winners.map(winner => `${playerIdToString(winner)}`)
+          );
+
+          addToHandHistory(
+            `The pot is split between ${winnerList}. Each player wins ${numberWithCommas(
+              win_amount
+            )}.`, //eslint-disable-line @typescript-eslint/camelcase
+            dispatch
+          );
+        }
+        // Else log an error in the console
+        else {
+          console.error(
+            "Incorrect winner ammount has been passed in to the log."
+          );
+        }
+      };
 
       const handleWinner = (): void => {
-        setWinner(message.winners[0], message.win_amount, state, dispatch);
-        addToHandHistory(
-          `Player${message.winners[0] + 1} wins ${message.win_amount}.`,
-          dispatch
-        );
+        setWinner(message.winners, message.win_amount, state, dispatch);
+        logWinners();
+        setTimeout(() => updateMainPot(win_amount, dispatch), 2000);
```

Comment:
It's because certain actions happen with a delay on the UI. After the winner is announced, there is a couple seconds delay before the chips are shipped to the winner. 

I'd love to know if there is a better way of achieving this.

Reply:
Thanks for your input!

Good to know that multiple `setTimeout`s can desynchronize. The delay is simply for "cosmetic" reasons, so I think we are fine for now. I will think about how we can structure the code so that multiple `setTimeout`s can be avoided.